assets/landscapes/Weather Report_files/index.js

```diff
@@ -0,0 +1,87 @@
+let initTemp = 71;
+const city = 'Seattle';
+
+document.getElementById('currentTemp').innerHTML = initTemp;
+
+cityValue = document.getElementById('inputCity').value = city;
+document.getElementById('city1').innerHTML = `${cityValue}✨`;
```

Comment:
In general it's best to have the global selections like this inside a [`document.addEventListener('DOMContentLoaded'` event handler](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event).

```suggestion
let cityValue = null;

window.addEventListener('DOMContentLoaded', (event) => {
  document.getElementById('currentTemp').innerHTML = initTemp;

  cityValue = document.getElementById('inputCity').value = city;
  document.getElementById('city1').innerHTML = `${cityValue}✨`;
});
```